common/driver/dma_buffer.c

```diff
@@ -106,12 +108,22 @@ size_t dmaAllocBuffers ( struct DmaDevice *dev, struct DmaBufferList *list,
 
       // Streaming buffer type, standard kernel memory
       else if ( list->dev->cfgMode & BUFF_STREAM ) {
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 15, 0)
+         buff->buffAddr = kmalloc(list->dev->cfgSize, GFP_KERNEL);
+         if (buff->buffAddr != NULL) {
+            buff->buffHandle = dma_map_single(list->dev->device, buff->buffAddr, list->dev->cfgSize, direction);
+            // Check for mapping error
+            if ( dma_mapping_error(list->dev->device,buff->buffHandle) ) {
+               buff->buffHandle = 0;
+            }
+         }
```

Comment:
Should add an `else` branch here and report error like in the `dma_alloc_pages` code below

Reply:
Add [49a0262](https://github.com/slaclab/aes-stream-drivers/pull/147/commits/49a02623cbdbacc54f9752eb17c2d18d82293efa)

Reply:
Is `kmalloc` ever expected to fail here? Should a message be printed if it does?
Thanks for adding the other `dev_err` though

Reply:
Added [f2212b5](https://github.com/slaclab/aes-stream-drivers/pull/147/commits/f2212b5b803a7138780cfdbc1e0922722a05926a)